tokenscan/scanner.go

```diff
@@ -0,0 +1,221 @@
+package tokenscan
+
+import (
+	"context"
+	"math"
+	"math/rand"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/gocql/gocql"
+
+	"github.com/scylladb/gocqlx"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/scylladb/gocqlx/qb"
+)
+
+const (
+	// parallelism = nodesInCluster * coresInNode * 3
+	defaultParallelism = 72
```

Comment:
?

Reply:
leftover from our setup, sorry. Anything sensible we can use here? Otherwise I'd make it a mandatory parameter in the constructor